package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rise-cache-v2",
-  "version": "1.6.4",
+  "version": "1.6.5",
```

Comment:
@rodrigopavezi You'll need to update this to `1.6.6` as my last PR was merged with `.5`, thanks